README.markdown

```diff
@@ -33,7 +33,13 @@ To install:
 
 and you should be set.
 
+### Throubleshooting
+
+Mac users may face problems when installing the package, it happens because on the installation build, xcon tries to import a module that is provided by OpenSSL, and most of the cases it fails because it can't find the module.
+What to do in these cases:
+ - Make sure openssl is installed `brew install openssl`
+ - Set an environment variable that tells the path to the openssl installed `export CFLAGS="-L/usr/local/opt/openssl/lib -I/usr/local/opt/openssl/include"`
```

Comment:
There's a couple more flags that may be needed. This is the output of `brew info openssl`:
```
ζ brew info openssl
openssl@1.1: stable 1.1.1g (bottled) [keg-only]
Cryptography and SSL/TLS Toolkit
https://openssl.org/
/usr/local/Cellar/openssl@1.1/1.1.1g (8,059 files, 18MB)
  Poured from bottle on 2020-07-16 at 18:17:09
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/openssl@1.1.rb
License: OpenSSL
==> Caveats
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  /usr/local/etc/openssl@1.1/certs

and run
  /usr/local/opt/openssl@1.1/bin/c_rehash

openssl@1.1 is keg-only, which means it was not symlinked into /usr/local,
because macOS provides LibreSSL.

If you need to have openssl@1.1 first in your PATH run:
  echo 'export PATH="/usr/local/opt/openssl@1.1/bin:$PATH"' >> ~/.zshrc

For compilers to find openssl@1.1 you may need to set:
  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"

For pkg-config to find openssl@1.1 you may need to set:
  export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
```

Reply:
Given that we don't really know, I would simply add a `(check brew info openssl)` at the end

Reply:
Great! I just added to it.